src/node_crypto.cc

```diff
@@ -5532,6 +5539,63 @@ void ExportChallenge(const FunctionCallbackInfo<Value>& args) {
   args.GetReturnValue().Set(outString);
 }
 
+
+// convert public key to compressed, uncompressed, hybrid format
+void ConvertKey(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+
+  CHECK_EQ(args.Length(), 3);
+
+  THROW_AND_RETURN_IF_NOT_BUFFER(args[0], "Public key");
+
+  size_t len = Buffer::Length(args[0]);
+  if (len == 0)
+    return args.GetReturnValue().SetEmptyString();
+
+  THROW_AND_RETURN_IF_NOT_STRING(args[1], "ECDH curve name");
+  node::Utf8Value curve(env->isolate(), args[1]);
+
+  int nid = OBJ_sn2nid(*curve);
+  if (nid == NID_undef)
+    return env->ThrowTypeError("Invalid ECDH curve name");
+
+  EC_GROUP* group = EC_GROUP_new_by_curve_name(nid);
+  if (group == nullptr)
+    return env->ThrowError("Failed to get EC_GROUP");
+
+  EC_POINT* pub = ECDH::BufferToPoint(env,
+                                      group,
+                                      Buffer::Data(args[0]),
+                                      len);
+  if (pub == nullptr)
+    return env->ThrowError("Failed to convert Buffer to EC_POINT");
```

Comment:
Leaks `group`.  `group` and `pub` are leaked on lines 5580 and 5587.  

Suggestion: use a std::unique_ptr with a custom deleter or use the std::shared_ptr trick from 8ccd320549.